libs/ledgerjs/packages/types-live/src/bridge.ts

```diff
@@ -51,32 +51,39 @@ export type PreloadStrategy = Partial<{
 /**
  *
  */
-export type BroadcastArg = {
-  account: Account;
+export type BroadcastArg<A> = {
```

Comment:
should A have some type constraints? (to be extending Account)

same question for the `SignOperation*`  types

Reply:
Sure I can 👌 
It's just that now they have been removed from usage outside of the definition of `AccountBridge` I think, but better be safe than sorry, so I'll update it 👍

Reply:
Done